tests/src/CLRTest.Execute.Batch.targets

```diff
@@ -229,9 +229,17 @@ IF NOT "%CLRTestExitCode%"=="%CLRTestExpectedExitCode%" (
 
     <PropertyGroup> 
       <_CLRTestRunFile Condition="'$(CLRTestIsHosted)'=='true'">"%CORE_ROOT%\corerun.exe"</_CLRTestRunFile>
+      <BatchCLRTestLaunchCmds><![CDATA[
+IF NOT "%CLRCustomTestLauncher%"=="" (
+  set LANUCHER=call %CLRCustomTestLauncher% %~dp0
+) ELSE (
+  set LANUCHER=%_DebuggerFullPath%
+)
+      ]]></BatchCLRTestLaunchCmds>
       <BatchCLRTestLaunchCmds Condition=" '$(IlasmRoundTrip)'=='true' "><![CDATA[
-ECHO $(_CLRTestRunFile) $(TargetAssemblyName) %CLRTestExecutionArguments% %Host_Args%
-$(_CLRTestRunFile) $(TargetAssemblyName) %CLRTestExecutionArguments% %Host_Args%
+$(BatchCLRTestLaunchCmds)
+ECHO %LAUNCHER% $(_CLRTestRunFile) $(TargetAssemblyName) %CLRTestExecutionArguments% %Host_Args%
```

Comment:
Basically what the old code was going after was this line after all the expansion:

```
call %CLRCustomTestLauncher% %~dp0 Finalizer.exe %CLRTestExecutionArguments% %Host_Args%
```

First argument to the launcher is the path, second is the EXE name, and arguments follow.